src/features/components/templates/hero-banners/trade/trade-hero.module.scss

```diff
@@ -17,10 +17,7 @@
         inline-size: 100%;
         background-color: $color-brand;
         inset-block-end: 0;
-        clip-path: polygon(0 28%, 100% 38%, 100% 0, 0 0);
-        @include rtl() {
-            clip-path: polygon(0 40%, 100% 60%, 100% 100%, 0% 100%);
-        }
+        clip-path: polygon(0 0, 100% 0, 100% 50%, 0 41%);
 
```

Comment:
changed because there was an issue in Arabic  mobile screen so I fixed it by removing these lines